### PR TITLE
Feat add app sketch and readme

### LINKED ISSUE #6: Description of your app & sketch

### PR DESCRIPTION
**Pull request by rezam747**
Added app sketch and updated readme

### CODE REVIEW

**Review by arijeetchatterjee — APPROVED** [review]

**Comment by mel-liow** [comment]
Thanks Reza! description is looking great! Would you be able to add a contributors and license section to the README.md too please?

File: README.md [hunk 1]
```diff
@@ -1 +1,27 @@
-# gapminder_dashboard
\ No newline at end of file
+# Gapminder dashboard
+
+## Decription of the app
+
+Gapminder data set is a great source to know about the world. So The main objective of this app is to investigate what the young population of students know and don’t know about basic global patterns and macro-trends and educate the intended users on global development trends. This dashboard has been developed to communicate insights about historical social/economic trends and create a better understanding of using and teaching statistics.
```
Comment (mel-liow) [inline README.md]: The Gapminder dataset is a great source of information to learn about the world. The main objective of this app is to investigate what the young population of students know and don’t know about basic global patterns and macro-trends and educate the intended users on global development trends. This dashboard has been developed to communicate insights about historical social/economic trends and create a better understanding of using and teaching statistics.

File: README.md [hunk 2]
```diff
@@ -1 +1,27 @@
...
+
+The landing page of the dashboard is designed to have five filters at the left, for selecting the target of the study, second factor of study, continent , country and the year range (1960 to 2018) .The line chart at the top is a trend-plot showing the change of the target of the study during the time over the years for the selected country, average of the continent and average of the whole world. So user would be able to compare these trend with each other. In the top right corner, there is a bubble chart which visualize the target study versus the second factor of study for all the countries.
```
Comment (mel-liow) [inline README.md]: The landing page of the dashboard is designed to have a panel of five filters for which users can interact with. These filters include selecting a primary and secondary target to study, a continent, a country and the year range of interest (1960 to 2018). The line chart is a trend-plot showing the change of the primary target of the study over the time interval for a selected country. It also includes the average of the continent and average of the whole world which gives users the ability to compare trends.
Additionally we include a bubble chart which visualizes the target study versus the second factor of study for all the countries.

File: README.md [hunk 3]
```diff
@@ -1 +1,27 @@
...
+
+The global map is presented at the bottom of the dashboard with tooltip for each country to show the target of the study data related to the selected countries. After choosing a continent in the filter section, just the selected continent stay colourful and all other turn to grey. The level of colour shading for selected continent would be based on the value of the target of study. To the right of the global map, there is a bar plot which shows the top ten countries with highest value for target of study.
```
Comment (mel-liow) [inline README.md]: The global map is presented at the bottom of the dashboard with a tooltip for each country. After choosing a continent in the filter section, the selected continent should be highlighted and the colour gradient will dictate the values of the target of study within that region. To the right of the global map, a bar plot will show the top ten countries with highest value for the selected target of study.

**Review by mel-liow — COMMENTED** [review]
Thanks @rezam747! Sorry, I've just made a few tweaks to correct some punctuation - hope this is ok!

File: README.md (hunk 3 above)
Comment (rezam747) [reply README.md]: Done👍🏻

File: README.md (hunk 2 above)
Comment (rezam747) [reply README.md]: Done👍🏻

File: README.md (hunk 1 above)
Comment (rezam747) [reply README.md]: Done👍🏻

**Pull request merged**